Src/BootCamp.Chapter/Checks.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Globalization; // NumberStyles.Float, CultureInfo.InvariantCulture
+
+class Lesson3
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Move 'Lesson3' into a named namespace.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4859834)